datalad/crawler/nodes/misc.py

```diff
@@ -589,3 +590,13 @@ def __call__(self, data):
             lgr.info("Ran node %s which yielded %d times", node, n)
             pdb.set_trace()
 
+
+def fix_url(data):
+    """Given data, get value within 'url' key and fix up so it is legit url
+
+    - replace spaces with %20
+    """
+
+    for key, value in data.items():
+         data[key] = quote(data[key], safe = ':/')
```

Comment:
- if I am correct in above, quote wouldn't work for us, but I could be wrong and @bpoldrack might provide better detail
- we do not want to "fix" all the values (note that you haven't even used `value` here) -- just the url field.  You could add an optional keyword argument to list the keys to check/fix (default to `['url']`)